samples/ldscripts/aarch64.ld

```diff
@@ -0,0 +1,15 @@
+/*
+Raspberry Pi does not have flash, but to accommodate for the picolibc.ld
```

Comment:
If this file is intended to be the same as raspi3b.ld then just INCLUDE that. If it's intended to vary independently then update the comment.

Reply:
Idea behind this approach is to have linker script for each variant. Just like with .cfg files. Indeed currently it is a copy of raspi3b.ld but may change in the future. More to say with naming in format ${variant}.ld there is no need to have mapping between variant and script file.